src/components/Header.tsx

```diff
@@ -37,7 +40,7 @@ function Header() {
     dispatch(fetchArticles(searchValue));
   }, [dispatch, searchValue]);
 
-  const handleSearchChange = (e) => {
+  const handleSearchChange = (e: object | any) => {
```

Comment:
Could you use a more descriptive parameter name for the `e` parameter? also, the change event has a specific type for their events, you can use that instead of object | any union type.